resources/i18n/ko/wiliwili.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title": "wiliwili",
+  "title": "윌리윌리",
```

Comment:
I think it's better to keep the original name of the software

Reply:
Ok, I'll change origin. :)